### PR TITLE
Feature/helix mrna

### PR DESCRIPTION
**Pull request by bputzeys**

### CODE REVIEW

File: docs/model_cards/helix_mrna.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+```
+
+## Contact
+support@helical-ai.com
+
+## Citation
+```bibtex
+@software{allard_2024_13135902,
+  author       = {Helical Team},
+  title        = {helicalAI/helical: v0.0.1-alpha6},
```
Comment (bputzeys) [inline docs/model_cards/helix_mrna.md:143]: will be alpha8 upon merge to release

File: _toc.yml [hunk 2]
```diff
@@ -6,6 +6,7 @@ root: docs/intro
 parts:
 - caption: Helical API Documentation 📚
   chapters:
+  - file: docs/Helix-mRNA.rst
```
Comment (bputzeys) [inline _toc.yml:9]: what about Mamba2-mRNA?

File: docs/model_cards/mamba2_mrna.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+```
+
+## Contact
+support@helical-ai.com
+
+## Citation
+```bibtex
+@software{allard_2024_13135902,
+  author       = {Helical Team},
+  title        = {helicalAI/helical: v0.0.1-alpha6},
```
Comment (bputzeys) [inline docs/model_cards/mamba2_mrna.md:124]: here too update version

**Comment by bputzeys** [comment]
Well done @mattwoodx !

**Review by mattwoodx — APPROVED** [review]

**Pull request merged**